release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2478,6 +2478,145 @@ index a90ab14ff..f7f6f1a71 100644
+
+*  Applying a `BMCEventSubscription` CR fails to create a Redfish events subscription. After the subscription YAML file is created and applied, no active Redfish subscription is visible. To workaround this issue, run the following command substituting appropriate BMC parameters to call the API directly and create the subscription:
++
+[source,terminal]
+----
+$ curl -X POST -i --insecure -u "<BMC_USERNAME>:<BMC_PASSWORD>" https://<BMC_IP>/redfish/v1/EventService/Subscriptions \
+-H 'Content-Type: application/json' \
+--data-raw '{
+"Protocol": "Redfish",
+"Context": "any string is valid",
+"Destination": "https://hw-event-proxy-openshift-hw-events.apps.example.com/webhook",
+}'
+----
++
+(link:https://issues.redhat.com/browse/OCPBUGSM-43707[*OCPBUGSM-43707*])
+
+* Using the GitOps ZTP pipeline to install a {sno} cluster in a disconnected environment, there should be two `CatalogSource` CRs applied in the cluster. One of the `CatalogSource` CRs gets deleted following multiple node reboots. As a workaround, you can change the default names, such as `certified-operators` and `redhat-operators`, of the catalog sources. (link:https://issues.redhat.com/browse/OCPBUGSM-46245[*OCPBUGSM-46245*])
+
+* A number of clusters fail to update during scale testing. After starting {product-title} 4.9.26 update with the Telecom vDU configuration applied, a cluster update to 4.10.13 initiated with the `ClusterVersion` CR fails at 15% and waits for cluster Operators updating to the target version. (link:https://issues.redhat.com/browse/OCPBUGSM-44655[*OCPBUGSM-44655*])
+
+* Spoke cluster deployment on bare-metal hosts using the converged workflow is blocked at inspection. An {rh-rhacm} 4.11 hub cluster using the `assisted-service` API gets stuck in an `inspecting` state, and eventually fails with an `Introspection` timeout. (link:https://issues.redhat.com/browse/OCPBUGSM-45428[*OCPBUGSM-45428*])
+
+* Sometimes, during ZTP GitOps pipeline {sno} installations, the Operator Lifecycle Manager registry-server container fails to reach the `READY` state. When creating subscriptions with a new `CatalogSource`, the `CatalogSource` remains in a `TRANSIENT_FAILURE` state. (link:https://issues.redhat.com/browse/OCPBUGSM-44041[*OCPBUGSM-44041*])
+
+* When you apply a tuned override to a pod and delete the tuned pod to force a restart, the pod should restart and the system should run normally. Instead a `systemd` hang can occur, causing the system to stop responding. (link:https://issues.redhat.com/browse/RHELPLAN-131021[*RHELPLAN-131021*])
+
+* When booting ZT system machines from a live ISO with static IPv6 address configuration, `NetworkManager` exits successfully before the interface link becomes ready, which leaves the network interface without configuration. As a workaround, edit the RHCOS ISO referenced in the `AgentServiceConfig` CR to append `rd.net.timeout.carrier` to the kernel parameters in the `grub.cfg` file:
```

Comment:
```suggestion
* When booting ZTP system machines from a live ISO with static IPv6 address configuration, `NetworkManager` exits successfully before the interface link becomes ready, which leaves the network interface without configuration. As a workaround, edit the RHCOS ISO referenced in the `AgentServiceConfig` CR to append `rd.net.timeout.carrier` to the kernel parameters in the `grub.cfg` file:
```

Reply:
I think you meant 'ZTP' here, yes?

Reply:
@tmalove Thanks for spotting this! ZT Systems is actually a computer manufacturer and some customers use their hardware and we're referring to that. I'll update it to `ZT Systems machines`